docs/api/SharedStyle.md

```diff
@@ -47,7 +47,7 @@ Create a new Shared Style with a specific name in a specific Document.
 
 A new SharedStyle
 
-## Create a new Instance
```

Comment:
Isn't this gone?

Reply:
that's why it's removed?

Reply:
👍 Oops, I must have been looking at an earlier commit before it was removed.